test/infrastructure/vcsim/controllers/images/images.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package images contains fake images for the vcsim content library.
+package images
+
+import (
+	_ "embed"
+)
+
+var (
+	// SampleOVF image.
```

Comment:
Do we have to use "real" files or would ~ empty ones also work?

Reply:
Dropped the biggest one, but I would leave the smallest one so we have something testing the code that uploads files to a content library.
Is that ok?